src/js/components/Pagination/Pagination.js

```diff
@@ -199,29 +201,75 @@ const Pagination = forwardRef(
       ...navProps[control],
     }));
 
-    return (
-      <StyledPaginationContainer
-        flex={false}
-        {...theme.pagination.container}
-        {...rest}
-      >
-        <Nav
-          a11yTitle={ariaLabel || a11yTitle || 'Pagination Navigation'}
-          ref={ref}
+    let content;
+
+    if (stepSelector) {
+      content = (
+        <Box
+          flex={false}
+          direction="row"
+          wrap
+          gap="small"
+          align="center"
+          justify="end"
+          stepSelectorLayoutProps
```

Comment:
What does `stepSelectorLayoutProps` do? it doesn't seem to have a value assigned.

Reply:
need to take out

Reply:
done